libs/javascript/@local/eslint-config/generate-ignore-patterns.cjs

```diff
@@ -0,0 +1,73 @@
+const path = require("node:path");
```

Comment:
This file is borrowed from BP repo. See https://github.com/blockprotocol/blockprotocol/pull/551 and planned changes in https://github.com/blockprotocol/blockprotocol/pull/680.